### PR TITLE
Update styling for the Add User form

### PR DESCRIPTION
**Pull request by mauberti-bc**
## Links to Jira Tickets

- N/A

## Description of Changes

- Updates styling on the Add User form for manual inserts
- Changed the form to handle only one user a time

## Testing Notes

- Should be able to add users

### CODE REVIEW

File: app/src/components/dialog/EditDialog.tsx [hunk 1]
```diff
@@ -124,7 +124,7 @@ export const EditDialog = <T extends FormikValues>(props: PropsWithChildren<IEdi
       initialValues={props.component.initialValues}
       enableReinitialize={true}
       validationSchema={props.component.validationSchema}
-      validateOnBlur={props.component.validateOnBlur ?? true}
+      validateOnBlur={props.component.validateOnBlur ?? false}
```
Comment (MacDeluca) [inline app/src/components/dialog/EditDialog.tsx]: Are we sure we want the default for this to be false?

**Review by MacDeluca — DISMISSED** [review]
🦖

File: app/src/components/dialog/EditDialog.tsx (hunk 1 above)
Comment (mauberti-bc) [reply app/src/components/dialog/EditDialog.tsx]: Yeah, I've heard complaints that its distracting - not a big deal either way

**Review by MacDeluca — APPROVED** [review]
🐕‍🦺

**Pull request merged**